engine/src/CalciteInterpreter.cpp

```diff
@@ -644,15 +672,25 @@ blazing_frame evaluate_split_query(std::vector<ral::io::data_loader> input_loade
 				blazing_timer.reset();
 
 				if(is_filtered_bindable_scan(query[0])) {
-					scan_frame.add_table(input_table);
-					// process_filter(queryContext, scan_frame, query[0]);
+					for (int i = 0; i < input_table.size(); ++i) {
```

Comment:
input_table is declared but has no size at declaration and nothing is added to it. This will never happen

Reply:
was a logic error that was fixed too thanks